pkg/eventlog/multireader.go

```diff
@@ -119,5 +120,5 @@ func exists(event Event, log []Event, checker storage.Checker) bool {
 		}
 	}
 
-	return checker.Exists(event.Module, event.Version)
+	return checker.Exists(context.Background(), event.Module, event.Version)
```

Comment:
Use `context.TODO()` here, as a placeholder until context is threaded through ReadSingle.

Reply:
I'm interested in what others think about this, they seem to return the same thing. Other than the name is there a reason to use one over the other?

Reply:
Done. No, `context.Background()` is more or less assumed to be correct, whereas `context.TODO()` is a placeholder to update later. Also, "TODO is recognized by static analysis tools that determine whether Contexts are propagated correctly in a program." https://golang.org/pkg/context/#TODO